core/sqf/sql/scripts/lobstart

```diff
@@ -30,8 +30,8 @@ let totalNodes=`grep 'Up' $tmp_node_status | wc -l`
 while [ ${count} -lt $totalNodes ]
 do
 sqshell -a > /dev/null << eof
-set {process \zlobsrv${count} } PERSIST_RETRIES=10,60
-set {process \zlobsrv${count} } PERSIST_ZONES=0,1
+set {process \$zlobsrv${count} } PERSIST_RETRIES=10,60
+set {process \$zlobsrv${count} } PERSIST_ZONES=${count}
 exec {name \$zlobsrv${count}, nowait, nid ${count}, out stdout_\$ZLOBSRV${count} } mxlobsrvr
```

Comment:
It looks like this lobstart script attempts to configure and start the lob servers in the nodes which are Up, while the change in sqgen configures it on all nodes. Let's say if we bring the cluster with one node down, the lobstart won't even configure the lob process on that node. When the node is reintegrated, will the lob processes come up. Also, my recommendation would be to configure the lobprocess in sqgen and let sqgen generate the lobstart script. That way, when the cluster is started with node down, the lob process would also be configured for the downed node and flagged as not available.